aten/src/ATen/native/ForeachUtils.h

```diff
@@ -128,7 +119,7 @@ bool can_use_fast_route(TensorList tensors, Scalar scalar) {
       return false;
     }
 
-    if (!will_promote_tensor(t, scalar)) {
+    if (will_promote_tensor(t, scalar)) {
```

Comment:
wait, was this computing the wrong thing before?

Reply:
no, the name was wrong